app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -244,6 +345,9 @@ impl Model {
 /// A widget for displaying the entries and name of a component group.
 ///
 /// The widget is rendered as a header label, a list of entries below it, and a colored background.
+/// It does not display the selection widget - because the selection jump between various component
+/// groups, the parent (Component List Panel) should own the selection widget; the Component Group
+/// View provide the information, where the widget should be placed when it's focused.
```

Comment:
- provide -> provides
- remove `,` before where
- Sometimes "Component Group" is written with upper case, sometimes with lower case

Reply:
Switched to upper case. BTW, the design doc is inconsistent too, however the upper case wins with over-10 to 3 score.